### PR TITLE
Update requirements.txt and add requirements_2024.5.txt

### PR DESCRIPTION
**Pull request by wgzintel**

### CODE REVIEW

File: tools/llm_bench/requirements.txt [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-openvino-tokenizers
-openvino_genai
+openvino~=2024.5.0
+openvino-tokenizers~=2024.5.0
+openvino_genai~=2024.5.0
 auto-gptq>=0.5.1 # for gptq
 pillow
 torch
-transformers>=4.40.0
+transformers>=4.46.2
```
Comment (peterchen-intel) [inline tools/llm_bench/requirements.txt:10]:
```suggestion
transformers~=4.46.0
```

**Review by peterchen-intel — APPROVED** [review]

File: tools/llm_bench/requirements_2024.5.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+numpy==2.0.2
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
+auto_gptq==0.7.1
+pillow==11.0.0
+torch==2.5.1+cpu
```
Comment (eaidova) [inline tools/llm_bench/requirements_2024.5.txt:6]:
```suggestion
torch==2.5.1
```

File: tools/llm_bench/requirements_2024.5.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+transformers==4.46.2
+diffusers==0.31.0
+openvino==2024.5.0
+#optimum is in dependency list of optimum-intel
```
Comment (eaidova) [inline tools/llm_bench/requirements_2024.5.txt:10]:
```suggestion
optimum==1.23.3
```

File: tools/llm_bench/requirements_2024.5.txt [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+git+https://github.com/huggingface/optimum-intel.git@d35737671ec226a66c495159588b7603066e1b07#egg=optimum-intel
+nncf ~= 2.14.0
+packaging==24.1
+psutil==6.1.0
+timm==1.0.11
+tiktoken==0.8.0
+onnx==1.16.1
```
Comment (eaidova) [inline tools/llm_bench/requirements_2024.5.txt:17]: for avoiding issues with dependabot
```suggestion
onnx==1.17.0
```

**Review by eaidova — APPROVED** [review]

**Comment by AlexKoff88** [comment]
The failed WWB tests were fixed in the main. I am ok with merging this PR as is.

**Pull request merged**